code/controller/controller.ino

```diff
@@ -62,80 +64,71 @@ void setup() {
 }
 
 void loop() {
-  bool is_pressed = false;
+  // bool is_pressed = false;  // unused variable?
```

Comment:
Yeah used to be used for print messages. You can delete lines and references to this variable.